app/src/i18n/locales/de/data.json

```diff
@@ -3,7 +3,7 @@
   "data-details": "Verbinden Sie Drittanbieterdaten oder laden Sie Ihre eigenen Daten hoch, um Ihre THG-Inventur nach der GPC-Basismethodik zu erstellen",
   "data-view-heading": "Sehen Sie die benötigten Daten, um die Inventur Ihrer Stadt abzuschließen",
   "data-view-details": "Die GPC-Basis umfasst drei Hauptsektoren: Stationäre Energie, Transport und Abfall. Geben Sie die notwendigen Daten für jeden Sektor ein, um eine umfassende THG-Inventur zu erstellen. <2>Weitere Informationen</2>",
-  "stationary-energy": "Stationäre Energie",
+  "stationary-energy": "Stationäre energie",
```

Comment:
Should be upper case in German like it was before (Energie as a noun is always upper case)